kpi/schema_extensions/v2/members/schema.py

```diff
@@ -0,0 +1,10 @@
+CHOICES = [
+    ('admin', 'Admin'),
+    ('member', 'Member'),
```

Comment:
Can we use already existing constants? `ORG_ADMIN_ROLE` and `ORG_MEMBER_ROLE`. BTW, I don't think the back end returns capitalized words . AFAIK, The API only returns the value of the constants.